WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -88,29 +105,114 @@ class PluginViewModel: Observable {
                 accessibilityIdentifier: "remove-plugin")
         }
 
+        var settingsLink: ImmuTableRow?
+        if let settingsURL = plugin.state.settingsURL, plugin.state.deactivateAllowed == true {
+            settingsLink = LinkRow(
+                title: NSLocalizedString("Settings", comment: "Link to plugin's Settings"),
+                action: { [unowned self] _ in
+                    self.presentBrowser(for: settingsURL)
```

Comment:
The settings link will likely end up in wp-login, since we don't have credentials to pre-authenticate Jetpack users. Maybe it's better not to show it until we can log them in automatically? @elibud thoughts?

Reply:
> until we can log them in automatically?
☝️ will we ever be able to do that on wp-admin?

Reply:
Hm — I think it's still worthwhile, since they can just tap to open it in Safari, where they could either be logged in or have the credentials stored in iCloud Keychain/1Password/etc. I know I do, but I guess it's probable that not many people do.

Reply:
I can't find that in Calypso, am I missing something or is it not there?

Reply:
@elibud here's Akismet as an example:
<img width="722" alt="sshot" src="https://user-images.githubusercontent.com/1594081/35113084-915f2db2-fc80-11e7-8797-d6fbf3ffce9a.png">

Reply:
Thanks, just tried that, we'll be replicating Calypso behavior and I'm fine with that. My uneducated guess is that any user that actually wants to modify the plugin settings and has good reason for it would have access or know her credentials.